src/briefcase/platforms/__init__.py

```diff
@@ -1,12 +1,11 @@
-try:
-    # Usually, the pattern is "import module; if it doesn't exist,
-    # import the shim". However, we need the 3.10 API for entry_points,
-    # as the 3.8 didn't support the `groups` argument to entry_points.
-    # Therefore, we try to import the compatibility shim first; and fall
-    # back to the stdlib module if the shim isn't there.
-    from importlib_metadata import entry_points
-except ImportError:  # pragma: no-cover-if-lt-py310
+import sys
+
+if sys.version_info >= (3, 10):  # pragma: no-cover-if-lt-py310
     from importlib.metadata import entry_points
+else:  # pragma: no-cover-if-gte-py310
+    # Before Python 3.10, entry_points did not support the group argument;
+    # so, the backport package must be used on older versions.
+    from importlib_metadata import entry_points
```

Comment:
I only updated this to be compatible with something that `pyupgrade` can understand and handle for us.

Reply:
Ack.